### ISSUE TITLE
Implement search page title fix

### ISSUE THREAD

**Issue by nichgray** [CDRH/lewisandclark#247]
Ideally we should get the search term into the page title. Review Orchid changes and see if they can be ported over.

**Comment by nichgray** [comment]
This should be fixed with https://github.com/CDRH/lewisandclark/pull/238/changes/684d8fe3a5ff75ca86fd5f4440b18ec05010b86b (and subsequent tinkering). These changes should be carefully reviewed.
